src/main.tsx

```diff
@@ -1,10 +1,18 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App.tsx'
-import './index.css'
+import React from "react";
+import ReactDOM from "react-dom/client";
+import LandingPage from "@pages/LandingPage";
```

Comment:
Won't `import { LandingPage } from "@pages/"` work? Purpose of path alias is so that you import from one place only, no need to explicitly specifiy file path.

Reply:
Thank you Sir Stan for the correction. I've pushed the corrected tsconfig file to accommodate the proper alias call. Thanks!